src/features/ajax/constants.js

```diff
@@ -0,0 +1,3 @@
+import { FEATURE_NAMES } from "../../loaders/features/features";
+
+export const FEATURE_NAME = FEATURE_NAMES.ajax
```

Comment:
BTW, I do think this model is onion-layering and confusing rather than just using FEATURE_NAMES.xyz directly in each aggregator file (that's already specific to each feature).

Reply:
imo we should try to isolate what is constant from those class files but these are subject and can be iterated on in the future for sure